java/java.graph/src/org/netbeans/modules/java/graph/NodeWidget.java

```diff
@@ -178,48 +184,52 @@ public String getConflictTooltip(GraphNode<I> node) {
             String version = scene.getVersion(firstConflict);
             String requester = parent != null ? parent.getName() : "???";
             tooltip.append(conflictType == VERSION_CONFLICT ? TIP_SingleConflict(version, requester) : TIP_SingleWarning(version, requester));
-        } else if (conflictCount > 1) {
-            tooltip.append(conflictType == VERSION_CONFLICT ? TIP_MultipleConflict() : TIP_MultipleWarning());
-            for (I nd : node.getDuplicatesOrConflicts()) {
-                if (scene.isConflict(nd)) {
-                    tooltip.append("<tr><td>");
-                    tooltip.append(scene.getVersion(nd));
-                    tooltip.append("</td>");
-                    tooltip.append("<td>");
-                    GraphNodeImplementation parent = nd.getParent();
-                    if (parent != null) {
+        } else {
+            if (conflictCount > 1) {
```

Comment:
Why did the "else if" become a separate "if" here?

Reply:
Fixed